### PR TITLE
[BEAM-10545] Advance the jupyterlab sidepanel extension version for first release

### PR DESCRIPTION
**Pull request by KevinGG**
Advance the version from 0.1.0 to 1.0.0

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [x] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [x] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Dataflow | Flink | Samza | Spark | Twister2
--- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Go/lastCompletedBuild/) | --- | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Go_VR_Flink/lastCompletedBuild/) | --- | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Go_VR_Spark/lastCompletedBuild/) | ---
Java | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Java/lastCompletedBuild/) | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow_Java11/lastCompletedBuild/) | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink_Java11/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Java_PVR_Flink_Batch/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Java_PVR_Flink_Streaming/lastCompletedBuild/) | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza/lastCompletedBuild/) | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Java_PVR_Spark_Batch/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_SparkStructuredStreaming/lastCompletedBuild/) | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Twister2/lastCompletedBuild/)
Python | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Python36/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Python37/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Python38/lastCompletedBuild/) | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Py_VR_Dataflow_V2/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | [Build Status](https://ci-beam.apache.org/job/beam_PreCommit_Python_PVR_Flink_Cron/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Python_VR_Flink/lastCompletedBuild/) | --- | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Python_VR_Spark/lastCompletedBuild/) | ---
XLang | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_XVR_Direct/lastCompletedBuild/) | --- | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_XVR_Flink/lastCompletedBuild/) | --- | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_XVR_Spark/lastCompletedBuild/) | ---

Pre-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

--- |Java | Python | Go | Website | Whitespace | Typescript
--- | --- | --- | --- | --- | --- | ---
Non-portable | [Build Status](https://ci-beam.apache.org/job/beam_PreCommit_Java_Cron/lastCompletedBuild/) | [Build Status](https://ci-beam.apache.org/job/beam_PreCommit_Python_Cron/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PreCommit_PythonLint_Cron/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PreCommit_PythonDocker_Cron/lastCompletedBuild/) <br>[Build Status](https://ci-beam.apache.org/job/beam_PreCommit_PythonDocs_Cron/lastCompletedBuild/) | [Build Status](https://ci-beam.apache.org/job/beam_PreCommit_Go_Cron/lastCompletedBuild/) | [Build Status](https://ci-beam.apache.org/job/beam_PreCommit_Website_Cron/lastCompletedBuild/) | [Build Status](https://ci-beam.apache.org/job/beam_PreCommit_Whitespace_Cron/lastCompletedBuild/) | [Build Status](https://ci-beam.apache.org/job/beam_PreCommit_Typescript_Cron/lastCompletedBuild/)
Portable | --- | [Build Status](https://ci-beam.apache.org/job/beam_PreCommit_Portable_Python_Cron/lastCompletedBuild/) | --- | --- | --- | ---

See [.test-infra/jenkins/README](https://github.com/apache/beam/blob/master/.test-infra/jenkins/README.md) for trigger phrase, status and link of all Jenkins jobs.

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by KevinGG** [comment]
R: @aaltay 
PTAL.

Let's talk offline to see if we need to create release branch and tags for a release.

**Comment by aaltay** [comment]
> R: @aaltay
> PTAL.
> 
> Let's talk offline to see if we need to create release branch and tags for a release.

Change LGTM. I am assuming you are planning to release this soon.

@kennknowles - Do we use branches or tags for releasing vendored dependencies? The release of this sub-directory will be similar to that. (I believe we do not do anything special other than a vote on dev list.)

**Comment by KevinGG** [comment]
Run Python PreCommit

**Comment by kennknowles** [comment]
I would just use a tag. The only purpose of a release branch is cherrypicking and continuing to do point releases.

**Comment by KevinGG** [comment]
> I would just use a tag. The only purpose of a release branch is cherrypicking and continuing to do point releases.

Thanks Kenn. 
@aaltay, could you please help me add a tag such as `jupyterlab-sidepanel-v1.0.0` when/after merging this PR?
I could not send a tag through a PR.

TL;DR: You could also use jlpm (yarn) to auto generate a tag:
```
jlpm config set version-tag-prefix "jupyterlab-sidepanel-v"
jlpm version  # Then input 1.0.0, it should auto generate a git tag for you.
```

Thanks!

**Comment by KevinGG** [comment]
Run Python PreCommit

**Comment by aaltay** [comment]
> > I would just use a tag. The only purpose of a release branch is cherrypicking and continuing to do point releases.
> 
> Thanks Kenn.
> @aaltay, could you please help me add a tag such as `jupyterlab-sidepanel-v1.0.0` when/after merging this PR?
> I could not send a tag through a PR.
> 
> TL;DR: You could also use jlpm (yarn) to auto generate a tag:
> 
> ```
> jlpm config set version-tag-prefix "jupyterlab-sidepanel-v"
> jlpm version  # Then input 1.0.0, it should auto generate a git tag for you.
> ```
> 
> Thanks!

I do not have jlpm. I guess I can clone the repo, and tag the specific commit and push. I do not know if there is an easier way to do this. Can I create a tag on github UI?

**Comment by KevinGG** [comment]
> > > I would just use a tag. The only purpose of a release branch is cherrypicking and continuing to do point releases.
> > 
> > 
> > Thanks Kenn.
> > @aaltay, could you please help me add a tag such as `jupyterlab-sidepanel-v1.0.0` when/after merging this PR?
> > I could not send a tag through a PR.
> > TL;DR: You could also use jlpm (yarn) to auto generate a tag:
> > ```
> > jlpm config set version-tag-prefix "jupyterlab-sidepanel-v"
> > jlpm version  # Then input 1.0.0, it should auto generate a git tag for you.
> > ```
> > 
> > 
> > Thanks!
> 
> I do not have jlpm. I guess I can clone the repo, and tag the specific commit and push. I do not know if there is an easier way to do this. Can I create a tag on github UI?

I think a tag is like a commit, you can use below command
```
git tag -fa jupyterlab-sidepanel-v1.0.0 -m "Extension: apache-beam-jupyterlab-sidepanel version 1.0.0" <this PR merge commit's hash>
git push origin(or upstream, whichever is the apache/beam remote) jupyterlab-sidepanel-v1.0.0 
```

**Comment by aaltay** [comment]
I merged this but I could not push the tag. I got 403 errors when I run `git push https://github.com/apache/beam jupyterlab-sidepanel-v1.0.0`.

I thought I should be able to do this because I did this for releases previously. @kennknowles - do you know what I might be doing wrong?

**Comment by aaltay** [comment]
Done: https://github.com/apache/beam/releases/tag/jupyterlab-sidepanel-v1.0.0

It worked using a personal access token instead of an ssh key (https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token#using-a-token-on-the-command-line). Not sure why.
